notes/force-move-implementation.md

```diff
@@ -0,0 +1,178 @@
+# ForceMove Implementation Notes
+
+Active set of notes explaining the ForceMove implementations.
+
+The goal is to make a close-to-optimal implementation of the force-move protocol on the EVM.
+
+## How to get the outcome from a finalized channel
+
+While channel settlement is out of the scope of this piece of work, we will need to interface with the channel funding and settlement layer at a later point.
+
+The strategy here will be to have a method that can be called when a channel is finalized, which will be responsible for interfacing. The precise details of this method will depend on the settlement layer. A few options here include (a) having an `assertOutcome` method which is passed an outcome and returns true if that is the outcome that was finalized or (b) having a `pushOutcome` method, which pushes the outcome into one or more other contracts in the settlement layer.
```

Comment:
This will depend a lot on how the settlement layer functions. Any of these methods would require a `bytes` outcome to be passed in and would start by computing the hash and doing an equality check. It's just a question on what's the most efficient thing to do next to allow funds to be withdrawn.